dir.props

```diff
@@ -13,8 +13,8 @@
   <PropertyGroup>
     <MajorVersion>2</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <PatchVersion>1</PatchVersion>
```

Comment:
This is the first release we have core-setup on BuildTools so I'm not sure if this is the only change needed or not. Can you please do a full build and look at the versions of the assets to make sure they all match?